subsys/bluetooth/services/fast_pair/fp_crypto/fp_crypto_oberon.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <errno.h>
```

Comment:
Is it actually used?

Reply:
added a missing dependency to fp_crypto.h which is needed for the basic types used in it. It was implicitly imported by this errno header